examples/com.eclipsesource.modelserver.example/src/main/java/com/eclipsesource/modelserver/example/ExampleModelServerModule.java

```diff
@@ -0,0 +1,17 @@
+package com.eclipsesource.modelserver.example;
+
+import java.util.Collection;
+
+import com.eclipsesource.modelserver.emf.configuration.EPackageConfiguration;
+import com.eclipsesource.modelserver.emf.di.EMFModelServerModule;
+
+public class ExampleModelServerModule extends EMFModelServerModule {
+
+	@Override
+	protected Collection<Class<? extends EPackageConfiguration>> bindEPackageConfigurations() {
+		Collection<Class<? extends EPackageConfiguration>> configs = super.bindEPackageConfigurations();
```

Comment:
I think we could improve the API here a bit by providing an additional empty template method in `EMFModelServerModule` which returns a collection of package configs to be registered (given that the Ecore package seems mandatory). This way clients would not be forced to call `super`.